fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -1724,4 +1724,11 @@ public String toMysqlDataTypeString() {
     public String toMysqlColumnTypeString() {
         return "unknown";
     }
+
+    // This function is called by Column::getMaxUniqueId()
+    // If type is a scalar type, it does not have field Id because scalar type does not have sub fields
+    // If type is struct type, it will return the max field id(default value of field id is -1)
```

Comment:
if type is array type ...
if type is map type (miss override)

Reply:
> if type is array type ... if type is map type (miss override)

done.